lib/tmdb_image_service.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module TmdbImageService
```

Comment:
This is our exciting new service that houses the api `/configuration` endpoint data and the url helper method.